modules/ngx_http_upstream_dyups_module/ngx_http_dyups_module.c

```diff
@@ -568,7 +568,15 @@ ngx_http_dyups_init_process(ngx_cycle_t *cycle)
     timer->log = cycle->log;
     timer->data = dmcf;
 
-    ngx_add_timer(timer, dmcf->read_msg_timeout);
+    /*
+     * when init process, break up timer, in case of shpool->mutex compete
+     */
+    srandom(ngx_pid);
+
+    delay = dmcf->read_msg_timeout > 1000 ? dmcf->read_msg_timeout : 1000;
+    t = ngx_random() % delay;
```

Comment:
use one variable for t and delay, they are the same usage. So we can only declare one var

Reply:
done